repository/src/main/java/com/michaldrabik/repository/TranslationsRepository.kt

```diff
@@ -33,28 +33,31 @@ class TranslationsRepository @Inject constructor(
   private val mappers: Mappers,
 ) {
 
-  fun getLanguage() = miscPreferences.getString(LANGUAGE, DEFAULT_LANGUAGE) ?: DEFAULT_LANGUAGE
+  fun getLocale() = miscPreferences.getString(LOCALE, null)?.let { AppLocale.fromCode(it) } ?: AppLocale.default()
```

Comment:
It would probably be cleaner to just get rid of this method from TranslationRepository and get locale from settings repository. Wherever TranslationRepository is injected.